docs/howto/operate/new-cluster/azure-or-google-cloud.md

```diff
@@ -26,8 +31,47 @@ Example `.tfvars` file:
 prefix     = "my-awesome-project"
 project_id = "my-awesome-project-id
 ```
+````
+
+````{tabbed} Azure
+The _minimum_ inputs this file requires are:
+
+- `subscription_id`: Azure subscription ID to create resources in.
+  Should be the id, rather than display name of the project.
+- `resourcegroup_name`: The name of the Resource Group that the cluster and other resources will be deployed into.
+- `global_container_registry_name`: The name of an Azure Container Registry to use for our image.
+  This must be unique across all of Azure.
+  You can use the following command to check your desired name is available:
+
+  ```bash
+  az acr check-name --name ACR_NAME --output table
+  ```
+
+- `global_storage_account_name`: The name of a storage account to use for Azure File Storage.
```

Comment:
Let's provide guidelines on naming conventions here? I'm not sure what the current names are, we can just pick that and riff off that?